lib/types/local-time.js

```diff
@@ -2,6 +2,8 @@
 const Long = require("long");
 const util = require("util");
 const utils = require("../utils");
```

Comment:
👀

Reply:
Why are them two of them?

Reply:
`utils` is file from DataStax in `lib\` directory, `util` is a node lib. These are different files with different functions.